cli/pcluster/utils.py

```diff
@@ -602,6 +602,23 @@ def get_master_ip_and_username(cluster_name):
     return master_ip, username
 
 
+def get_supported_archs_for_inst_type(instance_type):
```

Comment:
get_supported_archs_for_inst_type --> get_supported_instance_type_architectures ?

Reply:
I like the way it is better, because I think it makes it more clear what the functions doing. I'll remove the abbreviations, since I think that's what you don't like about it?

Reply:
ok for me 👍 thanks